samples/spring-customer-registry-views-quickstart/src/main/java/customer/api/Address.java

```diff
@@ -1,7 +1,4 @@
 package customer.api;
-/*
- * Copyright (C) 2009-2022 Lightbend Inc. <https://www.lightbend.com>
- */
```

Comment:
we usually don't add copyright headers on samples. And certainly not from 2009 for Kalix.